.env

```diff
@@ -1,7 +1,7 @@
 # Globals
 VERSION="0.0.2"
 CEPH_VERSION="17.2.6"
-SPDK_VERSION="23.01"
+SPDK_VERSION="23.01.1"
```

Comment:
Who/what is responsible for producing a new SPDK image and tagging it appropriately?

Reply:
@idryomov, see #180

Reply:
Do we need to wait until that is in place or can this PR be merged now?

Reply:
Thank you! I do not see dependency, I believe we could move forward.